src/xdocs/google_style.xml

```diff
@@ -2028,16 +2034,22 @@
                         src="images/ok_blue.png"
                         alt="" />
                   </span>
-                  <br />
                   <a href="checks/javadoc/singlelinejavadoc.html#SingleLineJavadoc">
                     SingleLineJavadoc</a>
+                  <br />
+                  <br />
+                  <span class="wrapper inline">
+                    <img
+                            src="images/ok_blue.png"
+                            alt="" />
+                  </span>
+                  <a href="checks/javadoc/invalidjavadocposition.html#InvalidJavadocPosition">
+                    InvalidJavadocPosition</a>
+                  <br />
                   Recent update for guide that clarify requirements will be
```

Comment:
This description belongs to `SingleLineJavadoc` but not to `InvalidJavadocPosition`

Reply:
done